pkg/build/gobuild.go

```diff
@@ -556,9 +559,10 @@ func createBuildArgs(buildCfg Config) ([]string, error) {
 }
 
 func (g *gobuild) configForImportPath(ip string) Config {
-	config, ok := g.buildConfigs[ip]
-	if !ok {
-		// Apply default build flags in case none were supplied
+	config := g.buildConfigs[ip]
+	if g.trimpath {
```

Comment:
I am trying to disable `-trimpath` for local debugging. After reading https://github.com/ko-build/ko/issues/500#issuecomment-970695696 I hoped that setting `flags: []` should be sufficient to do so.
Maybe, we could change the check in this line in the following way:
```go
	if g.trimpath && config.Flags == nil {
```
Or is there another way to make a build without trimpath that I am currently missing?

Reply:
`flags: []` should be sufficient I believe, or if not then you could set some other flag, such as `["-v"]`.
Did you verify that `ko` picked up the build config correctly from `.ko.yaml`?

Reply:
Yes, I verified that the .ko.yaml is used by adding an invalid flag that made the build fail. I also did a local build with a print statement and the trimpath flag was always appended no matter if other flags were set or not.